thunderbird/wps_io.py

```diff
@@ -0,0 +1,53 @@
+from pywps import LiteralInput, ComplexInput, ComplexOutput, FORMATS
+
+import logging
+
+
+def ncInput(identifier, title, abstract, max_occurs):
+    return ComplexInput(
+        identifier,
+        title,
+        abstract=abstract,
+        min_occurs=1,
+        max_occurs=max_occurs,
+        supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+    )
+
+
+def loglevel_input():
+    return LiteralInput(
+        "loglevel",
+        "Log Level",
+        default="INFO",
+        abstract="Logging level",
+        allowed_values=list(logging._levelToName.values()),
+    )
+
+
+def dryrun_input():
+    return LiteralInput(
+        "dry_run",
+        "Dry Run",
+        abstract="Checks file to ensure compatible with process",
+        data_type="boolean",
+    )
+
+
+def outFiles(abstract, formats):
+    return ComplexOutput(
+        "output",
+        "Output",
+        abstract=abstract,
+        as_reference=True,
+        supported_formats=formats,
+    )
```

Comment:
We may want to consider just make a full default here for metalink outputs.  The abstract can just be "Metalink output containing file links" or something along those lines. The format can be defaulted to `META4`. It would also help if the name had something to do with metalink.  The comment below also applies here.

Reply:
I originally thought of doing that, but [`wps_update_metadata.py` has the format as `netcdf` instead of `meta4`](https://github.com/pacificclimate/thunderbird/blob/375fb8aabc01e70409aa3870e5753b4e2b681b10/thunderbird/processes/wps_update_metadata.py#L54).

Reply:
I suppose we could just leave that specific output as is and change the other ones.

Reply:
> I suppose we could just leave that specific output as is and change the other ones.

Or we could define a netcdf output object! Either way works.